src/tshare/participant.rs

```diff
@@ -429,7 +437,7 @@ impl TshareParticipant {
             MessageType::Tshare(TshareMessageType::R2Decommit),
             decom,
         )?;
-        messages.extend_from_slice(&more_messages);
+        //messages.extend_from_slice(&more_messages);
```

Comment:
Minor change but this should be deleted with the next commit (I'm assuming it will but I'm just highlighting it so that it's not forgotten)